biztalk/core/feature-pack-create-application.md

```diff
@@ -0,0 +1,41 @@
+---
+title: Operation Name : Create BizTalk application
+description: In Azure DevOps, create a release definition to deploy the BizTalk Server application
+ms.custom: ""
+ms.date: "9/20/2022"
+ms.prod: "biztalk-server"
+ms.reviewer: ""
+
+ms.suite: ""
+ms.tgt_pltfrm: ""
+ms.topic: "article"
+author: MandiOhlinger
+ms.author: abkumar
+manager: "anneta"
+---
+
+
+# OperationName: Create BizTalk application
+
+
+### Create new BizTalk Server Appication

```

Comment:
This sub-heading is redundant (for all 3 pages)

Reply:
OR, have a single page for all 3 operations (since they don't have that much content), and keep subheadings for each operation